js/assessments/countWords.js

```diff
@@ -1,66 +1,79 @@
+			"The text contains %1$d words, this is more than the %2$d words recommended minimum.",
+			wordCount
+		);
+	}
 
-				//* translators: %1$d expands to the number of words in the text, %2$d to the recommended minimum of words */
-				text: i18n.dgettext( "js-text-analysis", "The text contains %1$d words, this is below the %2$d word recommended minimum. Add more useful " +
-					"content on this topic for readers." )
-			},
-			{
-				range: [ 0, 100 ],
-				score: -20,
+	if ( inRange( wordCount, 250, 300 ) ) {
+		score = 7;
 
-				/* translators: %1$d expands to the number of words in the text */
-				text: i18n.dgettext( "js-text-analysis", "The text contains %1$d words. This is far too low and should be increased." )
-			}
-		],
-		replacements: {
-			"%1$d": "%%result%%",
-			"%2$d": 300
-		}
-	};
-};
+		/* translators: %1$d expands to the number of words in the text, %2$s to the recommended minimum of words */
+		text = i18n.dngettext(
+			"js-text-analysis",
+			"The text contains %1$d word, this is slightly below the %2$d word recommended minimum. Add a bit more copy.",
+			"The text contains %1$d words, this is slightly below the %2$d words recommended minimum. Add a bit more copy.",
+			wordCount
+		);
+	}
 
-var countWordsAssessment = function( paper, i18n ) {
-	var result = 0;
+	if ( inRange( wordCount, 200, 250 ) ) {
+		score = 5;
 
-	if ( paper.hasText() ) {
-		result = countWords( paper.getText() );
+		/* translators: %1$d expands to the number of words in the text, %2$d to the recommended minimum of words */
+		text = i18n.dngettext(
+			"js-text-analysis",
+			"The text contains %1$d word, this is below the %2$d word recommended minimum. Add more useful content on this topic for readers.",
+			"The text contains %1$d words, this is below the %2$d words recommended minimum. Add more useful content on this topic for readers.",
+			wordCount
+		);
+	}
+
+	if ( inRange( wordCount, 100, 200 ) ) {
+		score = -10;
+
+		//* translators: %1$d expands to the number of words in the text, %2$d to the recommended minimum of words */
+		text = i18n.dngettext(
+			"js-text-analysis",
+			"The text contains %1$d word, this is below the %2$d word recommended minimum. Add more useful content on this topic for readers.",
+			"The text contains %1$d words, this is below the %2$d words recommended minimum. Add more useful content on this topic for readers.",
+			wordCount
+		);
+	}
+
+	if ( inRange( wordCount, 0, 100 ) ) {
+		score = -20;
+
+		/* translators: %1$d expands to the number of words in the text */
+		text = i18n.dngettext(
+			"js-text-analysis",
+			"The text contains %1$d word, this is far too low and should be increased.",
+			"The text contains %1$d words, this is far too low and should be increased.",
+			wordCount
+		);
 	}
 
-	var calculatedResult = new AssessmentResultCalculator( result, getScoringConfiguration( i18n ) );
+	text = i18n.sprintf( text, wordCount, 300 );
```

Comment:
Perhaps it would be clearer to move the recommended minimum `300` to a variable.